src/commands/stable-diffusion.ts

```diff
@@ -38,15 +38,24 @@ const generate: ICommandDefinition = {
 
 			cli.print(`[Stable Diffusion] Received prompt from ${message.from}: ${valueStr}`);
 
+			const huggingFaceAPIToken = process.env.HUGGINGFACE_API_TOKEN;
+
+			if (!huggingFaceAPIToken) {
+				cli.printError(`[Stable Diffusion] Huggingface API token not found, set the HUGGINGFACE_API_TOKEN environment variable`);
```

Comment:
I think we could add `message.reply` to notify the user as well, otherwise from whatsapp it's just failed silently

Reply:
Will add

Reply:
@Seitk Added